gradle/test.gradle

```diff
@@ -29,9 +49,11 @@ task guiTest(type: Test) {
     description = "Runs the acceptance GUI tests with Geb."
 }
 
-//TODO: smoke tests should be implemented as a task with a separate sourceSet (test-smoke)
-//please notice a known idea limitation relating to sourceSets: https://youtrack.jetbrains.com/issue/IDEA-138076
-task smokeTest(dependsOn: { guiTest })
+task smokeTest(type: Test) {
+    testClassesDir = sourceSets.smokeTest.output.classesDir
+    classpath = sourceSets.smokeTest.runtimeClasspath
+    systemProperties['smokeTestAppUrl'] = System.properties['smokeTestAppUrl']
+}
```

Comment:
You could configure also a separate reports path to not override base test reports. See [here](https://github.com/szpak/gradle-pitest-plugin/blob/master/build.gradle#L75-L77) for an example.